owslib/ogcapi/features.py

```diff
@@ -124,14 +124,23 @@ def collection_items(self, collection_id: str, **kwargs: dict) -> dict:
         @param filter: CQL TEXT expression
         @type cql: dict
         @param cql: CQL JSON payload
+        @type sortby: tuple
+        @param sortby: sort property and direction (`asc`, `desc`)
 
         @returns: feature results
         """
 
         if 'bbox' in kwargs:
             kwargs['bbox'] = ','.join(list(map(str, kwargs['bbox'])))
         if 'datetime_' in kwargs:
-            kwargs['datetime'] = kwargs['datetime_']
+            kwargs['datetime'] = kwargs.pop('datetime_')
+        if 'sortby' in kwargs:
+            sort_property, sort_order = kwargs.pop('sortby')
```

Comment:
Is there a risk of `ValueError: too many values to unpack (expected 2)` if a querystring is missing a value? `sortby=prop1`. Could a default be set for the sort order?